deps/rabbit/src/rabbit_disk_monitor.erl

```diff
@@ -161,11 +161,16 @@ handle_call({set_min_check_interval, MinInterval}, _From, State) ->
 handle_call({set_max_check_interval, MaxInterval}, _From, State) ->
     {reply, ok, set_max_check_interval(MaxInterval, State)};
 
-handle_call({set_enabled, _Enabled = true}, _From, State) ->
+handle_call({set_enabled, _Enabled = true}, _From, State = #state{enabled = true}) ->
     _ = start_timer(set_disk_limits(State, State#state.limit)),
-    rabbit_log:info("Free disk space monitor was enabled"),
+    rabbit_log:info("Free disk space monitor is already enabled"),
     {reply, ok, State#state{enabled = true}};
 
+handle_call({set_enabled, _Enabled = true}, _From, State = #state{enabled = false}) ->
+  _ = start_timer(set_disk_limits(State, State#state.limit)),
+  rabbit_log:info("Free disk space monitor was manually enabled"),
+  {reply, ok, State#state{enabled = true}};
+
 handle_call({set_enabled, _Enabled = false}, _From, State) ->
```

Comment:
What do you think about expanding these `false` cases like the above `true` cases so that the messages are slightly better?

```erlang
handle_call({set_enabled, _Enabled = false}, _From, State = #state{enabled = false}) ->
    %% ... says that the monitor was already disabled
handle_call({set_enabled, _Enabled = false}, _From, State = #state{enabled = true}) ->
    %% ... disables the monitor
```

Reply:
Good idea!